php/test/test.php

```diff
@@ -78,90 +99,84 @@ function test_order_book ($exchange, $symbol) {
 
 //-----------------------------------------------------------------------------
 
-function test_trades ($exchange, $symbol) {
-
+function test_trades($exchange, $symbol) {
     if ($exchange->has['fetchTrades']) {
-
         $delay = $exchange->rateLimit * 1000;
-        usleep ($delay);
-
-        dump (green ($symbol), 'fetching trades...');
-        $trades = $exchange->fetch_trades ($symbol);
-        dump (green ($symbol), 'fetched', green (count ($trades)), 'trades');
+        usleep($delay);
 
+        dump(green($symbol), 'fetching trades...');
+        $trades = $exchange->fetch_trades($symbol);
+        dump(green($symbol), 'fetched', green(count($trades)), 'trades');
     } else {
-
-        dump (green ($symbol), 'fetchTrades () not supported');
+        dump(green($symbol), 'fetchTrades () not supported');
     }
 }
 
-function test_symbol ($exchange, $symbol) {
-    test_ticker ($exchange, $symbol);
-    if ($exchange->id == 'coinmarketcap')
-        dump (var_export ($exchange->fetchGlobal ()));
-    else
-        test_order_book ($exchange, $symbol);
+function test_symbol($exchange, $symbol) {
+    test_ticker($exchange, $symbol);
+    if ('coinmarketcap' == $exchange->id) {
```

Comment:
Why would they switch left and right in every comparison... I'm thinking left to right, and the left is the variable we are comparing, and on the right is the value we are comparing to... heh...

Reply:
Nothing about the ordering of values inside the `if`-clause here: https://pear.php.net/manual/en/standards.control.php ...